templates/hooks/db_instance/sdk_create_post_set_output.go.tpl

```diff
@@ -0,0 +1,9 @@
+	// We expect the DB instance to be in 'creating' status since we just
+	// issued the call to create it, but I suppose it doesn't hurt to check
+	// here.
+	if instanceCreating(&resource{ko}) {
+		// Setting resource synced condition to false will trigger a requeue of
+		// the resource. No need to return a requeue error here.
+		setSyncedCondition(&resource{ko}, corev1.ConditionFalse, nil, nil)
+		return &resource{ko}, nil
+	}
```

Comment:
Looking at these, i think we should do the same for `dynamodb-controller` instead of returning a "requeueError"

Reply:
yeah, we don't want to return a requeue error in the ReadOne code path for things like that. Just adding a ConditionTypeResourceSynced condition with value of ConditionFalse is enough to trigger a requeue due to this code:

https://github.com/aws-controllers-k8s/runtime/blob/86654532edbd5e5cf99d7f89c2aa561bce810f14/pkg/runtime/reconciler.go#L610-L616

in the main reconciler.